.github/scripts/cygwin/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+source `dirname ${BASH_SOURCE[0]}`/../config.sh
+
+CYGWIN_BUILD_PATH=$BUILD_PATH/cygwin
```

Comment:
the same idea for build target

Reply:
Here it's not relevant as the `$BUILD_PATH` already contains target string.